docs/Inputs.md

```diff
@@ -891,6 +891,13 @@ const PersonEdit = () => (
 );
 ```
 
+**Reminder:** [react-hook-form's `formState` is wrapped with a Proxy](https://react-hook-form.com/api/useformstate/#rules) to improve render performance and skip extra computation if specific state is not subscribed. So, make sure you deconstruct or read the `formState` before render in order to enable the subscription.
```

Comment:
This one is weird as we don't use `useFormState` anywhere before or did I miss something?

Reply:
We use the `formState` (`formState: { isSubmitted }`) so I figured I'd rather add the Reminder here anyway. But I'm open to discussion :grin: